### ISSUE TITLE
Cannot read NaT timestamps from .hyper extracts

### ISSUE THREAD

**Issue by dingobar** [innobi/pantab#56]
**Describe the bug**

Writing dataframe with NaT timestamps dotted arount to hyper works, but reading it back gives an out of bounds timestamp 1-01-01 00:00:00 for those fields, which throws an error.
![image](https://user-images.githubusercontent.com/41419288/71244276-6d65e800-2312-11ea-860b-79d46ce6bbd8.png)

**To Reproduce**

```python
import pandas as pd
import pantab
from pathlib import Path
import numpy as np
from datetime import datetime
from tableauhyperapi import TableName

np.random.seed(0)
df = pd.DataFrame(
    {"A": np.random.rand(10), "B": np.random.rand(10)}, index=range(10)
)
df["datetime"] = datetime.now()
df.iloc[1] = None

pantab.frame_to_hyper(df, Path("rng.hyper"), table=TableName("rng"))
pantab.frame_from_hyper("rng.hyper", table=TableName("rng")) # Throws the error
```

**Expected behavior**
Should be able to these out of bounds timestamps as NaT in dataframe.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Windows

**Additional context**

**Comment by WillAyd** [comment]
Thanks for the report! Is this an issue with version 0.2.0? Wonder if downgrading to 0.1.1 temporarily fixes - if so I have an idea where things went wrong

**Comment by WillAyd** [comment]
Hmm never mind - downgrading shouldn't help

**Comment by WillAyd** [comment]
Should be fixed with the latest version 0.2.1 so just `python -m pip install --upgrade pantab` and should be good to go
